config/manifests/patches/spec-descriptors.yaml

```diff
@@ -0,0 +1,452 @@
+        - urn:alm:descriptor:com.tectonic.ui:text
+    - displayName: Rack Node Affinity Labels
+      description: NodeAffinityLabels to pin the rack, using node affinity
+      path: racks[0].nodeAffinityLabels
+      x-descriptors:
+        - 'urn:alm:descriptor:com.tectonic.ui:nodeAffinity'
+    - path: racks[0].zone
+      description: |
+        failure-domain.beta.kubernetes.io/zone node label value
+        where pods related to this rack will be restricted. Note
+        the value may be in the topology.kubernetes.io/zone label
+        as well.
+      displayName: Zone
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+    - path: dseWorkloads.analyticsEnabled
+      description: |
+        Enables the analytics advanced workload on DSE pods
+      displayName: Analytics
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:booleanSwitch
+        - urn:alm:descriptor:com.tectonic.ui:hidden
+    - path: dseWorkloads.searchEnabled
+      description: |
+        Enables the search advanced workload on DSE pods
+      displayName: Search
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:booleanSwitch
+    - path: dseWorkloads.graphEnabled
+      description: |
+        Enables the graph advanced workloads on DSE pods
+      displayName: Graph
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:booleanSwitch
+        - urn:alm:descriptor:com.tectonic.ui:hidden
+    - path: resources
+      description: "Resource requests and limits for each Cassandra pod in the \ncluster. Note: Only specify CPU and memory limits here.\n"
+      displayName: Resources
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:resourceRequirements
+    - path: networking.hostNetwork
+      description: |
+        Enables host network configuration for pods. Note only one pod is permitted per worker in this configuration.
+      displayName: Host network
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:booleanSwitch
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+    - path: networking.nodePort
+      description: |
+        Exposed ports on the Kubernetes node level for forwarding services to Cassandra pods. Most deployments will not use this functionality.
+      displayName: Node Port Network Configuration
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+    - path: networking.nodePort.internode
+      description: |
+        Internode port to use when leveraging Node Port network configuration.
+      displayName: Internode Port
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+    - path: networking.nodePort.internodeSSL
+      description: |
+        Internode SSL port to use when leveraging Node Port network configuration.
+      displayName: Internode SSL Port
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
```

Comment:
Let's use the `urn:alm:descriptor:com.tectonic.ui:number` descriptor instead of text.

Reply:
```suggestion
        - urn:alm:descriptor:com.tectonic.ui:number
```